tsconfig.json

```diff
@@ -18,8 +18,7 @@
     "src/**/*.ts",
     "!node_modules/**",
     "custom_typings/*.d.ts",
-    "typings/main/*.d.ts",
```

Comment:
Including each definition file itself seems to be unnecessary, since the exact file paths are written as references to `typings/index.d.ts`